<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Irene-PC" name="Coto" tests="1" failures="1" timestamp="3 Apr 2015 13:46:15 GMT" time="3.481" errors="0">
  <testcase name="@AfterTest addScreenshot" time="0.0" classname="services.coto.CotoBot">
    <failure type="java.lang.IllegalArgumentException" message="wrong number of arguments">
      <![CDATA[java.lang.IllegalArgumentException: wrong number of arguments
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest addScreenshot -->
  <testcase name="@AfterTest closeDriver" time="0.0" classname="services.coto.CotoBot">
    <skipped/>
  </testcase> <!-- @AfterTest closeDriver -->
  <testcase name="run" time="0.156" classname="services.coto.CotoBot">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
	at services.coto.CotoBot.run(CotoBot.java:21)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- run -->
</testsuite> <!-- Coto -->
